subhub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -1,11 +1,17 @@
+import time
+import json
+
 import mockito
 import requests
 import boto3
+from botocore.exceptions import ClientError
```

Comment:
group the from imports separate from the regular imports

Reply:
I had not heard of this separation guideline before but I implemented.